erdpy/cli_contracts.py

```diff
@@ -235,7 +250,10 @@ def _prepare_contract(args: Any) -> SmartContract:
 
 
 def _prepare_sender(args: Any) -> Account:
```

Comment:
shouldn't the function return Union[Account, LedgerAccount] instead of Account ?

Reply:
removed the Union as now Account is the base class of the LedgerAccount